chainbase/src/main/java/org/tron/core/db2/core/SnapshotImpl.java

```diff
@@ -83,6 +99,20 @@ public void merge(Snapshot from) {
     Streams.stream(fromImpl.db).forEach(e -> db.put(e.getKey(), e.getValue()));
   }
 
+  public void mergeFullData(Snapshot from) {
```

Comment:
As I see, the `merge` method is used for merging the `snapshotImpl` which is behind itself,  this method `mergeFullData` is used for merging the `snapshotImpl` which is ahead, and only one `snapshot`. it only can merge all data by another outside iteration.

So renaming the method name to `mergeAhead` or `mergeBefore` or `mergeFront` may be better?

Reply:
OK, I renamed the mergeFullData function to mergeAhead